SoftLayer/managers/ordering.py

```diff
@@ -370,6 +373,21 @@ def get_price_id_list(self, package_keyname, item_keynames):
 
         return prices
 
+    @staticmethod
+    def get_item_price_id(core, price, price_id):
+        """get item price id"""
+        category_code = []
```

Comment:
is `category_code` used for anything? I don't see it being returned, if not just remove it.

Reply:
Yes, the category_code is used, is where I store the category code data for each item to be able to get just one item because there a lot of items with the same category code.